main/integration-tests/__test__/api-tests/studies/__fixtures__/update-study-fixture.js

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const BaseFixture = require('../../../../helpers/base-fixture');
+const { getInternalUserToken } = require('../../../../utils/auth-tokens');
+const { createStudyJson, createStudy } = require('../../../../utils/studies');
+const { createUserJson, createUser } = require('../../../../utils/users');
+
+// Test Fixture
+/**
+ * This class is designed to pre-generate & help generate (using methods) resources
+ * that would be required by a SPECIFIC integration test suite
+ *
+ * The verification of these components are performed once every respective test suite cycle
+ * For example, whenever update-study-api test suite is triggered
+ */
+class UpdateStudyFixture extends BaseFixture {
```

Comment:
Nit: Can we remove the word update from the class name and file name for update-study-fixture.js and update-study-api.test.js? Once we start filling in integ tests, other study API tests will be added in here right?

Reply:
Well, not necessarily. Depending on how many more tests are needed for the update study suite, and what the requirements for other study scenarios are, we might need to create test suites accordingly. 
Having said that, these test suites should be broad enough to not cause confusion (eg. ONLY one test suite for ALL study update scenarios).

The idea is that `main/integration-tests/__test__/api-tests/studies` will house all study-related test suites. Test fixtures need to be kept as independent as possible.